internal/pkg/secureboot/measure/internal/pcr/bank_data_test.go

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package pcr_test
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/stretchr/testify/require"
+
+	"github.com/siderolabs/talos/internal/pkg/secureboot"
+	"github.com/siderolabs/talos/internal/pkg/secureboot/measure/internal/pcr"
+	talostpm2 "github.com/siderolabs/talos/internal/pkg/tpm2"
+)
+
+func TestCalculateBankData(t *testing.T) {
+	t.Parallel()
+
+	pemKey, err := os.ReadFile("../../testdata/pcr-signing-key.pem")
```

Comment:
embed maybe?

Reply:
I think in this case it probably makes less sense, as it's shared across multiple tests, wdyt?

we also use it in other places as a file path, so we to be consistent I'm not embedding it

Reply:
yup, makes sense